tests/integration/schema/test_configs.py

```diff
@@ -20,24 +22,54 @@
 models_config_path = config_path / "models"
 datasets_config_path = config_path / "datasets"
 experiments_config_path = config_path / "experiments"
+augmentation_config_path = config_path / "augmentations"
 
 models_config_files = [[item] for item in models_config_path.rglob("*.yaml")]
 datasets_config_files = []
-experiment_config_files = [[item] for item in experiments_config_path.rglob("*.yaml")]
+experiment_config_files = [
+    [item] for item in experiments_config_path.rglob("*.yaml")
+]
+augmentation_config_files = []
+augmentation_keys = (
+    []
+)  # List of values from ['color', 'combined', 'position', 'postprocessing', 'preprocessing']
+for item in os.listdir(augmentation_config_path):
+    if os.path.isfile(item):
+        continue
+    files = [cfg for cfg in (augmentation_config_path / item).rglob("*.yaml")]
```

Comment:
it could be rewritten as: 
`files = list((augmentation_config_path / item).rglob("*.yaml"))`